main/nav_backplane_viewer.py

```
@@ -0,0 +1,1366 @@
+
+    def _on_body_selection_changed(self) -> None:
+        name = self._body_combo.currentText()
+        if not name:
+            self._compose_and_display()
+            return
+        # Load prefs if any
+        prefs = self._body_prefs.get(name)
+        if prefs:
+            self._body_alpha.blockSignals(True)
+            self._body_cmap.blockSignals(True)
+            self._body_alpha.setValue(int(round(float(prefs.get('alpha', 0.5)) * 100)))
```

Ruff (0.14.5)

Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)